### DISCUSSION TITLE
v1.4.0

### DISCUSSION THREAD

**Discussion by asvae** [epicmaxco/vuestic-ui#1798] [General]
Took all our elbow grease, but it's finally there 🤗.

## Exciting stuff
* [Nuxt 3 support](https://vuestic.dev/en/getting-started/nuxt)
* [Carousel component](https://vuestic.dev/en/ui-elements/carousel)
* [Counter component](https://vuestic.dev/en/ui-elements/counter)

## Less exciting stuff
* Full transition to composition API
* IDE typing support for components
* betterer SSR support (vite, vue-ssr, nuxt)

## Possible next steps
* Accessability
* Layouts

<hr /><em>This discussion was created from the release <a href='https://github.com/epicmaxco/vuestic-ui/releases/tag/'>v1.4.0</a>.</em>

**Comment by prostoleo** [comment]
Фрейм выглядит круто, не спорю. Но что у вас с сайтом? Он постоянно не доступен, то вылетает 404, то просто не открывается. Чере VPN тоже пробовал - тоже самое.

**Reply by m0ksem** [reply]
> Фрейм выглядит круто, не спорю. Но что у вас с сайтом? Он постоянно не доступен, то вылетает 404, то просто не открывается. Чере VPN тоже пробовал - тоже самое.

👋, какой именно сайт и когда? Я таких проблем не наблюдал.
